### PR TITLE
pubsub: rework URL comments for consistency

### PR DESCRIPTION
**Pull request by vangent**
Updates #1174.

### CODE REVIEW

File: pubsub/azurepubsub/azurepubsub.go [hunk 1]
```diff
@@ -18,10 +18,13 @@
 //
 // URLs
 //
-// For pubsub.OpenTopic/Subscription URLs, azurepubsub registers for the scheme
-// "azuresb". The Service Bus Connection String defaults to the environment
-// variable "SERVICEBUS_CONNECTION_STRING". For details on the format of the
-// URL, see URLOpener.
+// For pubsub.OpenTopic and pubsub.OpenSubscription, azurepubsub registers
+// for the scheme "azuresb".
+// The default URL opener will use a Service Bus Connection String based on
```
Comment (shantuo) [inline pubsub/azurepubsub/azurepubsub.go]: Probably in another PR, I think the package should be renamed to `servicebus` to be consistent.

File: pubsub/azurepubsub/azurepubsub.go [hunk 2]
```diff
@@ -117,8 +118,10 @@ const Scheme = "azuresb"
 //   - The URL's host+path is used as the topic name.
```
Comment (shantuo) [inline pubsub/azurepubsub/azurepubsub.go]: The URL above should be `azuresb://mytopic?subscription=`

File: pubsub/azurepubsub/azurepubsub.go (hunk 1 above)
Comment (vangent) [reply pubsub/azurepubsub/azurepubsub.go]: You are probably right. I'm not sure it's worth it at this point though. `awspubsub` should probably be something more like `snssqs` too.

On the other hand, I think it's nice to have `azure` in there to make it more recognizable.

Let's discuss at standup tomorrow.

File: pubsub/azurepubsub/azurepubsub.go (hunk 2 above)
Comment (vangent) [reply pubsub/azurepubsub/azurepubsub.go]: Ooh, good catch. Done.

**Comment by vangent** [comment]
PTAL

**Pull request merged**